pkg/starboard/config.go

```diff
@@ -208,6 +213,10 @@ type BuildInfo struct {
 	Date    string
 }
 
+type Config interface {
```

Comment:
We don't want to have such generic config because we plan to have more configuration options. Each scanner should define it's set of configuration settings.